src/enkf/gridio_gfs.f90

```diff
@@ -355,6 +447,7 @@ subroutine readgriddata_pnc(vars3d,vars2d,n3d,n2d,levels,ndim,ntimes, &
   end if
 
   ! cloud derivatives
+  use_full_hydro = .true.
   if (.not. use_full_hydro .and. iope==0) then
```

Comment:
Add comment and fix indentation of if block that follows.

Reply:
Done!